tools/js2c.py

```diff
@@ -278,6 +279,10 @@ def JS2C(source, target):
         split = split[1:]
       name = '/'.join(split)
 
+    # if its a gypi file we're going to want it as json
+    # later on anyway, so get it out of the way now
+    if name.endswith(".gypi"):
+      lines = json.dumps(eval(lines))
```

Comment:
There is code for similar purposes in `tools/install.py` and it only does:

https://github.com/nodejs/node/blob/a8b5192fefe54b140ae923e749104b49c5d3be49/tools/install.py#L23-L24

Maybe we can just do the same so don't have to `eval` the config here..

Reply:
imo eval is cleaner, what if (for some ungodly reason) a config value has an escaped quote in it.

Reply:
It does the right thing.  `config.gypi` is produced by `pprint.pformat()` and always uses single quotes for strings.  `\'hola\'` becomes `\"hola\"` and that's legal JSON.